stdlib/2and3/xml/sax/saxutils.pyi

```diff
@@ -1,12 +1,12 @@
 import sys
-from typing import AnyStr, Mapping
+from typing import AnyStr, Mapping, Text, Union
 
 from xml.sax import handler
 from xml.sax import xmlreader
 
-def escape(data: AnyStr, entities: Mapping[str, str] = ...) -> AnyStr: ...
-def unescape(data: AnyStr, entities: Mapping[str, str] = ...) -> AnyStr: ...
-def quoteattr(data: AnyStr, entities: Mapping[str, str] = ...) -> AnyStr: ...
+def escape(data: AnyStr, entities: Mapping[Union[str, Text], Union[str, Text]] = ...) -> AnyStr: ...
```

Comment:
The return type isn't quite right anymore. On my machine, I see (in Python 2):
```
>>> from xml.sax import saxutils
>>> saxutils.escape('a', {'a': u'd'})
u'd'
>>> saxutils.escape('a', {u'a': 'd'})
u'd'
```
So the return type doesn't necessary match the type of `data`.

Reply:
good point! so i suppose most generally all types should be Union[str, Text]] even though that loses us a bit of information in the cases where AnyStr was correcy

Reply:
I think you can just used `Text`; `Union[str, Text]` is redundant in Python 3, and in Python 2, `Text` should be used for something that can be either `bytes` or `unicode` (https://github.com/python/typeshed/blob/master/CONTRIBUTING.md).